src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
```

Comment:
> To be compatible with Mono we should do nothing

@danmosemsft, I think this is a dangerous stance.  If no apps use it, Mono compatibility doesn't matter.  If apps do use it, they're expecting this security-focused feature to prevent something that it's not going to prevent.